src/modules/AutoinstScripts.rb

```diff
@@ -7,6 +7,34 @@
 require "yast"
 
 module Yast
+
+  # Module responsible for autoyast scripts support.
+  # See Autoyast Guide Scripts section for user documentation and some parameters explanation
```

Comment:
Perhaps a link would be handy.

Reply:
do you have link for that for master? I found only for specific SLE or Leap version. Maybe TW one exists?

Reply:
I am not aware of any TW documentation.

Reply:
The latest released SLE docs is: https://documentation.suse.com/sles/15-SP1/html/SLES-all/configuration.html#createprofile-scripts

Reply:
@mvidner yep, but I would like to link ideally always the master....so maybe https://doc.opensuse.org/projects/autoyast/#id-1.9.5.34.10.4 is better link?

Reply:
That generated numeric anchor will break. https://doc.opensuse.org/projects/autoyast/#createprofile-scripts is better